specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

```diff
@@ -630,6 +630,78 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/purge": {
+      "post": {
+        "description": "Purges data in an Log Analytics workspace by a set of user-defined filters.",
+        "operationId": "Workspace_Purge",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspacePurgeParameter"
+          }         
+        ],
+        "responses": {
+          "202": {
```

Comment:
What is the final response when the PURGE operation succeeds?

Reply:
It should be modeled in the 200 response

Reply:
Moved 200 from purge status to here, added x-ms-long-running-operation